### DISCUSSION TITLE
Welcome to DiscordBotClient Discussions!

### DISCUSSION THREAD

**Discussion by aiko-chan-ai** [aiko-chan-ai/DiscordBotClient#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nipica206** [comment]
**THIS IS ABSOLUTE GOLD**
I once used a client that was exactly like this (jk this is way better) but sadly they stopped working on it, and since then I've been searching for another one that does it's job at the best and I finally found it!

KEEP UP THE GOOD WORK!!!

**Comment by aiko-chan-ai** [comment]


**Reply by aiko-chan-ai** [reply]
looks like it will cause api spam

**Comment by JMcrafter26** [comment]
I love this thing! 10/10

**Reply by aiko-chan-ai** [reply]
Thank you
<img src='https://media.discordapp.net/stickers/1012664855083364352.png'>
